src/studio/src/designer/backend/Designer.csproj

```diff
@@ -26,7 +26,7 @@
     <PackageReference Include="ini-parser-netstandard" Version="2.5.2" />
     <PackageReference Include="JWTCookieAuthentication" Version="2.4.2" />
     <!-- Do not upgrade this package unless extensively tested-->
-    <PackageReference Include="LibGit2Sharp" Version="0.27.0-preview-0034" />
```

Comment:
I'm guessing this isn't required anymore since LibGit2Sharp should be updated allready?